py/src/braintrust/wrappers/genai/__init__.py

```diff
@@ -0,0 +1,443 @@
+import logging
+import time
+from typing import Any, Dict, Iterable, List, Optional, Tuple
```

Comment:
nit: should we call this `google_genai`? `genai` is slightly loaded e.g. `genai.*` opentelemetry stuff.